src/Database/Mysql/Driver/Mysql.php

```diff
@@ -17,17 +17,25 @@
 namespace Tenancy\Database\Drivers\Mysql\Driver;
 
 use Illuminate\Database\ConnectionInterface;
-use Illuminate\Database\QueryException;
+use Illuminate\Support\Facades\App;
 use Illuminate\Support\Facades\DB;
 use Tenancy\Affects\Connections\Contracts\ResolvesConnections;
 use Tenancy\Database\Drivers\Mysql\Concerns\ManagesSystemConnection;
 use Tenancy\Facades\Tenancy;
+use Tenancy\Hooks\Database\Contracts\ProcessesQueries;
 use Tenancy\Hooks\Database\Contracts\ProvidesDatabase;
 use Tenancy\Hooks\Database\Events\Drivers as Events;
 use Tenancy\Identification\Contracts\Tenant;
 
 class Mysql implements ProvidesDatabase
 {
+    protected $queryManager;
+
+    public function __construct()
+    {
+        $this->queryManager = App::make(ProcessesQueries::class);
+    }
```

Comment:
Do we really want to do it like this and not simply let it resolve from the container in the construct?

Reply:
You can't, because the driver class isn't resolved by the container.  It's `new`d directly in the ResolvesDatabase (I think...?) listener

Reply:
You could app make in that listener in that case? 🤔